monster_waves/RandomGenerator.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+#include <random>
+
+class RandomNumber
+{
+public:
+    RandomNumber();
+
+    template<typename T>
+    T getNumber(T minNumber, T maxNumber);
+private:
+    std::mt19937 mt;
+};
+
+template<typename T>
+
```

Comment:
@JakubMastalski niepotrzebna przerwa, przez to nie wiadomo czego tyczy się szablon...
- [ ] usunąć niepotrzebną przerwę